### PR TITLE
#11959 apply autotyping and commit test modules that pass mypy

### LINKED ISSUE #11959: apply autotyping to tests and commit those that pass mypy
**Is your feature request related to a problem? Please describe.**
most test modules consist of hundreds of `-> None:` test functions that could be automatically typed with https://github.com/JelleZijlstra/autotyping/ in --safe mode

**Describe the solution you'd like**
apply https://github.com/JelleZijlstra/autotyping/ and commit any test modules that pass mypy

### PR DESCRIPTION
**Pull request by graingert**
## Scope and purpose

Fixes #11959

### CODE REVIEW

File: src/twisted/web/test/test_flatten.py [hunk 1]
```diff
@@ -632,6 +632,7 @@ def test_traceback(self) -> None:
         If a L{FlattenerError} is created with traceback frames, they are
         included in the string representation of the exception.
         """
+
```
Comment (adiroiban) [inline src/twisted/web/test/test_flatten.py:635]: We can leave this change.
Was this change also generated by auto-typing ?

Maybe a bug in auto-typing and upstream might want to know about it.

File: src/twisted/web/test/test_template.py [hunk 2]
```diff
@@ -217,7 +217,6 @@ def test_file(self) -> None:
 
 
 class XMLLoaderTestsMixin:
-
```
Comment (adiroiban) [inline src/twisted/web/test/test_template.py:220]: Was this change also generated by auto-typing ?

Maybe a bug in auto-typing and upstream might want to know about it.

**Review by adiroiban — APPROVED** [review]
Looks good. Many thanks.

The missing coverage is ok.

Only a small comment.

Feel free to merge when you have time. Thanks again.

File: src/twisted/web/test/test_flatten.py (hunk 1 above)
Comment (graingert) [reply src/twisted/web/test/test_flatten.py:635]: autotyping runs via libcst which automatically applies black, this change is from that

File: src/twisted/web/test/test_template.py (hunk 2 above)
Comment (graingert) [reply src/twisted/web/test/test_template.py:220]: autotyping runs via libcst which automatically applies black, this change is from that

**Pull request merged**